gossip/src/restart_crds_values.rs

```diff
@@ -29,6 +29,25 @@ pub enum RestartLastVotedForkSlotsError {
     LastVotedForkEmpty,
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, AbiExample, Debug)]
+pub struct RestartHeaviestFork {
+    pub from: Pubkey,
+    pub wallclock: u64,
+    pub last_slot: Slot,
+    pub last_slot_hash: Hash,
+    // Sum of received heaviest fork validator stake / total stake * u16::MAX.
+    received_heaviest_fork_ratio: u16,
+    pub shred_version: u16,
+}
+
+#[derive(Debug, Error, PartialEq)]
+pub enum RestartHeaviestForkError {
+    #[error("Received stake larger than total stake")]
+    ReceivedHeaviestForkStakeLargerThanTotalStake,
+    #[error("Total stake of received heaviest fork cannot be zero")]
+    ReceivedHeaviestForkStakeZero,
```

Comment:
No need to repeat the enum name in the variants.
The variants are already qualified by the enum name, ie. `RestartHeaviestForkError::...`.

Reply:
Changed.